### ISSUE TITLE
Colour contrast on sorting arrows fails WCAG 2.2 non text contrast (2.41:1)

### ISSUE THREAD

**Issue by MelanieTPX** [tpximpact/mhclg-oruk#189]
The sorting arrows in the table headers on https://staging.openreferraluk.org/developers/dashboard) has insufficient colour contrast ratio, failing to meet the WCAG 2.2 Level AA (Minimum) requirement for Success Criterion 1.4.11 Non-Text Contrast.

Non-Text Element (Arrow): #9CA3AF

Background Colour: #FCF9F5

Contrast Ratio: 2.41:1

Required Ratio (WCAG AA for Non-Text): 3:1

Expected outcome - can we update the colour of the sorting arrow to achieve a contrast ratio of at least 3:1 against the background please?

<img width="1500" height="986" alt="Image" src="https://github.com/user-attachments/assets/934c3f78-0e26-4487-87a9-f2532e7c6bb1" />
 
<img width="2564" height="526" alt="Image" src="https://github.com/user-attachments/assets/4e333d1b-9049-4e07-9628-585cc3bf70f2" />
